src/libstore/local-store.cc

```diff
@@ -751,7 +751,7 @@ void LocalStore::registerDrvOutput(const Realisation & info, CheckSigsFlag check
     if (checkSigs == NoCheckSigs || !realisationIsUntrusted(info))
         registerDrvOutput(info);
     else
-        throw Error("cannot register realisation '%s' because it lacks a valid signature", info.outPath.to_string());
+        throw Error("cannot register realisation '%s' because it lacks a trustworthy signature", info.outPath.to_string());
```

Comment:
```suggestion
        throw Error("cannot register realisation '%s' because it lacks a signature by a trusted public key", info.outPath.to_string());
```